includes/common.php

```diff
@@ -427,27 +425,6 @@ function del_dev_attrib($device, $attrib_type)
     return DeviceCache::get((int) $device['device_id'])->forgetAttrib($attrib_type);
 }
 
-/*
- * convenience function - please use this instead of 'if ($debug) { echo ...; }'
- */
-if (! function_exists('d_echo')) {
```

Comment:
This is used a bit here and there

Reply:
@Jellyfrog it is a duplicate function. There is still an implementation.